### PR TITLE
Moved all m and s cluster charts to 6.5 Milestone builds

### PR DESCRIPTION
**Pull request by wajihahmed**

### CODE REVIEW

File: samples/config/prod/m-cluster/common.yaml [hunk 1]
```diff
@@ -6,14 +6,14 @@ fqdn: openam.prod.frk8s.net
...
-#  tag: 6.5.0-M0
+#  tag: 6.5.0-M7
 #  pullPolicy: Always
 
 # Note if you enable TLS then you have to either provide the certificate 
 # by running generate-tls.sh script or you have to have cert-manager deployed
 # for your DNS domain
  
-tlsStrategy: https-cert-manager    
+tlsStrategy: https
```
Comment (dgoldssfo) [inline samples/config/prod/m-cluster/common.yaml]: Doesn't this go away now? Pretty sure it's no longer necessary.

File: samples/config/prod/m-cluster/configstore.yaml [hunk 2]
```diff
@@ -1,11 +1,14 @@
 image:
   #repository: forgerock-docker-public.bintray.io/ds
   repository: gcr.io/engineering-devops/ds
-  tag: 6.5.0-M0
+  tag: 6.5.0-M129.2
   pullPolicy: Always
   
 instance: configstore
 
+cts:
```
Comment (dgoldssfo) [inline samples/config/prod/m-cluster/configstore.yaml]: What is this? (For my information, never saw it before. Is it something I need to add to the DevOps Guide?)

File: samples/config/prod/m-cluster/ctsstore-local.yaml [hunk 3]
```diff
@@ -1,27 +1,38 @@
 image:
   #repository: forgerock-docker-public.bintray.io/ds
   repository: gcr.io/engineering-devops/ds
-  tag: 6.5.0-M129.1-TTL-SL
+  tag: 6.5.0-M129.2-CTS-TTL
   pullPolicy: Always
 
 instance: ctsstore-local
 
+userstore:
```
Comment (dgoldssfo) [inline samples/config/prod/m-cluster/ctsstore-local.yaml]: And this?

File: samples/config/prod/m-cluster/ctsstore-local.yaml [hunk 4]
```diff
@@ -1,27 +1,38 @@
...
-   -server -Xms11g -Xmx11g -XX:+UseCompressedOops -XX:+UseG1GC -XX:+UseNUMA 
-   -XX:MaxGCPauseMillis=100 -verbose:gc -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps 
-   -XX:+PrintGCDetails -XX:+PrintPromotionFailure -XX:+PrintAdaptiveSizePolicy -Xloggc:/tmp/gc.log
+  -server -Xms11g -Xmx11g -XX:+UseG1GC -XX:+UseNUMA -XX:+UseCompressedOops
+  -XX:MaxGCPauseMillis=200 -XX:G1MixedGCCountTarget=12 -XX:MetaspaceSize=256m  
+  -XX:ConcGCThreads=5 -XX:+PrintGCApplicationStoppedTime
+  -XX:+ParallelRefProcEnabled -XX:InitiatingHeapOccupancyPercent=70
+  -verbose:gc -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+PrintGCDetails
+  -XX:+PrintPromotionFailure -XX:+PrintAdaptiveSizePolicy -XX:+PrintTenuringDistribution 
+  -Xloggc:/tmp/gc.log
```
Comment (dgoldssfo) [inline samples/config/prod/m-cluster/ctsstore-local.yaml]: Wow :-)

File: samples/config/prod/m-cluster/configstore.yaml (hunk 2 above)
Comment (wajihahmed) [reply samples/config/prod/m-cluster/configstore.yaml]: This has always been there in the original charts values.yaml.  I am only exposing it now in m-cluster.

File: samples/config/prod/m-cluster/ctsstore-local.yaml (hunk 3 above)
Comment (wajihahmed) [reply samples/config/prod/m-cluster/ctsstore-local.yaml]: This has always been there in the original charts values.yaml.  I am only exposing it now in m-cluster.

**Pull request merged**